doc/appendix-de-kassensichv/receipt-case-definitions/receipt-case-definitions.md

```diff
@@ -11,14 +11,18 @@ The issuance and redemption of vouchers are processes with DSFinV-K requirements
 
-The issuance of single-purpose vouchers is covered by our charge item cases (`ftChargeItemCase`): `0x4445000000000061`, `0x4445000000000062`, `0x4445000000000063`, `0x4445000000000064`, `0x4445000000000065`, `0x4445000000000066` and `0x4445000000000067`. 
+The issuance of single-purpose vouchers is covered by our charge item cases (`ftChargeItemCase`): `0x4445000000000061`, `0x4445000000000062`, `0x4445000000000063`, `0x4445000000000064`, `0x4445000000000065`, `0x4445000000000066` and `0x4445000000000067`.
 
-The redemption of single-purpose vouchers is covered by `ftChargeItemCase`: `0x4445000000000069`, `0x444500000000006A`, `0x444500000000006B`, `0x444500000000006C`, `0x444500000000006D`, `0x444500000000006E`, `0x444500000000006F`. When used in the request, the charge item amount of the good or service should be a positive value and the charge item amount of the voucher should be negative (e.g. +1 and -1).
+The redemption of single-purpose vouchers is covered by `ftChargeItemCase`: `0x4445000000000069`, `0x444500000000006A`, `0x444500000000006B`, `0x444500000000006C`, `0x444500000000006D`, `0x444500000000006E`, `0x444500000000006F`. When used in the request, the charge item amount of the good or service should be a positive value and the charge item amount of the voucher should be negative (e.g. +15 and -15).
 
-If the cash register can not handle the redemption of single-purpose vouchers within the charge items than it can alternatively use a pay item having the pay item case (`ftPayItemCase`): `0x444500000000000A`. When used like this in the request, the charge item amount of the good or service should be a positive value and the pay item amount of the voucher should also be a positive value (e.g. +1 and +1).
+If the cash register can not handle the redemption of single-purpose vouchers within the charge items than it can alternatively use a pay item having the pay item case (`ftPayItemCase`): `0x444500000000000A`. When used like this in the request, the charge item amount of the good or service should be a positive value and the pay item amount of the voucher should also be a positive value (e.g. +15 and +15).
+
+Please find examples of requests with single-purpose vouchers [here](../examples/vouchers.md#single-purpose-voucher-issuance).
 
 The issuance of multi-purpose vouchers is covered within the pay items by `ftPayItemCase`: `0x444500000000000D`. Negative amounts get converted to an issuance. Since there is no specific product or service coverded by the voucher, the issuance is not taxable and no VAT rate must be specified. Same applies for redemption since the VAT rate is included in the charge items for the products or services of the request. For redemption of a multi-purpose voucher, one can use `ftPayItemCase`: `0x444500000000000D` with a positive amount. 
 
 If the cash register can not handle the redemption within a pay item, then it can alternatively use a charge item having `ftChargeItemCase`: `0x4445000000000068`. 
```

Comment:
If the cash register can not handle the redemption within a pay item, then it can alternatively use a charge item having `ftChargeItemCase`: `0x4445000000000068` with negative amount.

Reply:
Done, thank you!